README.md

```diff
@@ -1,5 +1,31 @@
-# MPES Reader
+[![Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff)
+![](https://github.com/FAIRmat-NFDI/pynxtools-mpes/actions/workflows/pytest.yml/badge.svg)
+![](https://github.com/FAIRmat-NFDI/pynxtools-mpes/actions/workflows/pylint.yml/badge.svg)
+![](https://github.com/FAIRmat-NFDI/pynxtools-mpes/actions/workflows/publish.yml/badge.svg)
+![](https://img.shields.io/pypi/pyversions/pynxtools-mpes)
+![](https://img.shields.io/pypi/l/pynxtools-mpes)
+![](https://img.shields.io/pypi/v/pynxtools-mpes)
+![](https://coveralls.io/repos/github/FAIRmat-NFDI/mpes/badge.svg?branch=main)
+
+# A reader for MPES data
+
+## Installation
+
+It is recommended to use python 3.11 with a dedicated virtual environment for this package.
+Learn how to manage [python versions](https://github.com/pyenv/pyenv) and
+[virtual environments](https://realpython.com/python-virtual-environments-a-primer/).
+
+This package is a reader plugin for [`pynxtools`](https://github.com/FAIRmat-NFDI/pynxtools) and thus should be installed together with `pynxtools`:
 
-## Contact person in FAIRmat for this reader
-Florian Dobener
 
+```shell
+pip install pynxtools[mpes]
+```
+
+for the latest development version.
+
+## Docs
+Extensive documentation of this pynxtools plugin is available [here](https://fairmat-nfdi.github.io/pynxtools-mpes/). You can find information about getting started, how-to guides, the supported file formats, how to get involved, and much more there.
+
+## Contact person in FAIRmat for this reader
+Lukas Pielsticker, Laurenz Rettig
```

Comment:
Should we add contact infos here as well?

Reply:
I think we didn't want to do that in the repos because of email address scrapers and spam. We can just put a link to https://www.fair-di.eu/fairmat/about-fairmat/team-fairmat